geometry_msgs/PoseStamped object_pose_world
---
bool success
